pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -54,63 +55,77 @@ func TestFullClusterBackup(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	settings := clustersettings.MakeTestingClusterSettings()
-	params := base.TestClusterArgs{
-		ServerArgs: base.TestServerArgs{
-			// Disabled only because backupRestoreTestSetupEmpty, another DR test
-			// helper function, that is not yet enabled to set up tenants within
-			// clusters by default. Tracking issue
-			// https://github.com/cockroachdb/cockroach/issues/76378
-			DefaultTestTenant: base.TODOTestTenantDisabled,
-			Settings:          settings,
-			Knobs: base.TestingKnobs{
-				SpanConfig: &spanconfig.TestingKnobs{
-					// We compare job progress before and after a restore. Disable
-					// the automatic jobs checkpointing which could possibly mutate
-					// the progress data during the backup/restore process.
-					JobDisablePersistingCheckpoints: true,
-				},
-				GCJob: &sql.GCJobTestingKnobs{
-					// We want to run the GC job to completion without waiting for
-					// MVCC GC.
-					SkipWaitingForMVCCGC: true,
-				},
-			},
-		}}
-	const numAccounts = 10
-	_, sqlDB, tempDir, cleanupFn := backupRestoreTestSetupWithParams(t, singleNode, numAccounts, InitManualReplication, params)
-	tcRestore, sqlDBRestore, cleanupEmptyCluster := backupRestoreTestSetupEmpty(t, singleNode, tempDir, InitManualReplication, params)
-	defer cleanupFn()
-	defer cleanupEmptyCluster()
+	// It is easier to assert state in this test if both the backing up and
+	// restoring cluster are run with the same tenant option. We already have
+	// targeted tests for combinations of backing up and restoring between a
+	// system and application tenant.
+	var runOnceUnderDuress bool
+	testutils.RunTrueAndFalse(t, "inSystemTenant", func(t *testing.T, inSystemTenant bool) {
```

Comment:
do we need to run this test under both configs for every test run? I believe each case would take many seconds to run. I don't feel strongly about this though.

Reply:
I added logic to run it once under duress cause as you said this takes multiple minutes. Under non-duress runs thought its not too bad, and probably worth the extra coverage.